packages/google_maps_flutter/google_maps_flutter_web/pubspec.yaml

```diff
@@ -35,3 +35,9 @@ dev_dependencies:
 # The example deliberately includes limited-use secrets.
 false_secrets:
   - /example/web/index.html
+  
+# TODO: Remove ovveride before merge
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides: 
+  google_maps_flutter_platform_interface:
+    path: ../google_maps_flutter_platform_interface
```

Comment:
With the fallback you added in the platform interface that `animateCameraWithDuration` by default calls `animateCamera`, you don't need to touch anything on the Web implementation. You could revert all these files and have one package less to deal with.

(If you were overriding this to run web tests: thank you!)